modules/dnn/test/test_darknet_importer.cpp

```diff
@@ -125,12 +125,14 @@ class Test_Darknet_nets : public DNNTestLayer
                           const std::vector<int>& refClassIds,
                           const std::vector<float>& refConfidences,
                           const std::vector<Rect2d>& refBoxes,
-                          double scoreDiff, double iouDiff, float confThreshold = 0.24)
+                          double scoreDiff, double iouDiff, float confThreshold = 0.24,
+                          int batch_size = 1)
```

Comment:
Please swap `confThreshold` and `batch_size` arguments because now all the tests put `batch_size = 5` in the wrong variable.